server/game/cards/03_TWI/units/BattleDroidEscort.ts

```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class BattleDroidLegion extends NonLeaderUnitCard {
```

Comment:
The tests caught this - wrong class name

Reply:
ok, that is embarrassing, fixed up